clients/feeder/feeder_test.go

```diff
@@ -311,7 +311,7 @@ func TestBlockWithSequencerAddressUnmarshal(t *testing.T) {
 	assert.Equal(t, "0x5dcd266a80b8a5f29f04d779c6b166b80150c24f2180a75e82427242dab20a9", block.SequencerAddress.String())
 }
 
-func TestClassUnmarshal(t *testing.T) {
+func TestClassV0Unmarshal(t *testing.T) {
```

Comment:
Could we use [this](https://github.com/starkware-libs/cairo/blob/main/crates/cairo-lang-starknet/test_data/hello_starknet.json) to test v1 classes?

Reply:
Since that is not real data from a network, it might be hard to put it together with our existing test data. I thought https://github.com/NethermindEth/juno/pull/638/files#diff-a6ae96f1fa557bcf68f6d1e27521660124c4c1bbcaa8d46193ec26f7047245adR51 would be enough, since it uses the client already. Let me know what you think

Reply:
done, https://github.com/NethermindEth/juno/pull/638/files#diff-ada8ad9f7998cd8be2d874e9605538aa33f4c2a0cfe7b8d16f921743073c1207R337